app/stores/toplevel/ProfileStore.js

```diff
@@ -1,11 +1,12 @@
 // @flow
-import { observable, computed } from 'mobx';
+import { observable, computed, action } from 'mobx';
 import BigNumber from 'bignumber.js';
 import moment from 'moment/moment';
 import Store from '../base/Store';
 import Request from '../lib/LocalizedRequest';
 import environment from '../../environment';
-import { THEMES } from '../../themes';
+import { THEMES } from '../../types/ThemeType';
```

Comment:
it’s strange that is all caps. If we switch `Themes` to `Theme`, we could change `THEMES` to `Themes`

Reply:
@nicarq 
was not sure,
`THEMES` is a constant
`Themes` is a `type`
changing `Themes` to `Theme` has no doubt.